web/ai_service.py

```
@@ -0,0 +1,141 @@
+from django.conf import settings
+import google.generativeai as genai
+from openai import OpenAI
+import requests
+import re
+import markdown
+from pygments import highlight
+from pygments.lexers import get_lexer_by_name
+from pygments.formatters import HtmlFormatter
+
+class AIService:
+    def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)